web/app/themes/mitlib-parent/inc/site-search.php

```diff
@@ -20,18 +20,14 @@
 		</div>
 	</form>
 
-	<h2>Browse our <a href="/site-search">A-Z index of pages</a> on this site.</h2>
-
-	<p>You can also check out these commonly-used resources:</p>
+	<p>Need help? <a href="/ask">Ask us!</a></p>
 
+	Or try:
```

Comment:
Hmm, interesting - I was going to ask that this text be enclosed in a tag, but the W3C validator doesn't flag it as a problem. So nevermind :-)

Reply:
I think it's probably part of a div at some point so it's _technically_ fine. However, I'm glad you pointed it out as now it's bugging me and I suspect I'll throw it in a <p> if it doesn't muck up the display too much.